src/main.rs

```diff
@@ -62,6 +65,11 @@ fn main() {
             println!("hermit git is not yet implemented");
         }
         ("init", Some(matches)) => {
+            let name = match matches.value_of("SHELL_NAME") {
+                Some(shell_name) => shell_name,
+                None => "default",
+            };
+            init::new_profile(name.to_string());
```

Comment:
Can we pass a `&str` into `new_profile` and get rid of this `to_string` call?  It's unnecessary allocation.